examples/erlang/esp32/ap_network.erl

```diff
@@ -0,0 +1,38 @@
+-module(ap_network).
+
+-export([start/0]).
+
+start() ->
+    Config = [
+        {ap, [
+            % {ssid, esp:nvs_get_binary(atomvm, ap_ssid, <<"myssid">>)},
+            {psk,  esp:nvs_get_binary(atomvm, ap_psk, <<"!!MVmota">>)},
```

Comment:
I think you've hardcoded here some password, please, remove it ;)

Reply:
That was actually kind deliberate, because the absence of a password means, create an open network.  How about this -- I can leave both commented out, with "myssid" and "mypsk", and I can add a comment that without an SSID the atomvm-mac is created, and without a password, the network is open (not recommended)

Reply:
Commented out, added comment and changed to "mypsk"